config/robots.rb

```diff
@@ -1474,11 +1474,11 @@
     }
   )
 
-  custom_robot('nx-96-lhr-cherrypick-to-lhr-xp',
-               name: 'NX-96 LHR Cherrypick => LHR XP',
+  custom_robot('nx-96-lhr-rt-cherrypick-to-lhr-xp',
```

Comment:
Maybe `nx-96-lhr-rt-to-lhr-xp` not `nx-96-lhr-rt-cherrypick-to-lhr-xp`

Reply:
yep good spot will fix